src/spikeinterface/extractors/neoextractors/neuronexus.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from spikeinterface.core.core_tools import define_function_from_class
+
+from .neobaseextractor import NeoBaseRecordingExtractor, NeoBaseSortingExtractor
+
+
+class NeuroNexusRecordingExtractor(NeoBaseRecordingExtractor):
+    """
+    Class for reading data from NeuroNexus Allego.
+
+    Based on :py:class:`neo.rawio.NeuronexusRawIO`
+
+    Parameters
+    ----------
+    file_path : str
```

Comment:
This can take path as well, right?

Reply:
Yes because we cast it to a path to resolve. I can add that :)